Packs/GoogleChatViaWebhook/Integrations/GoogleChatViaWebhook/GoogleChatViaWebhook.yml

```diff
@@ -76,33 +62,17 @@ script:
     description: Send a customizable card to Google Chat Space via Incoming Webhook.
     name: send-google-chat-custom-card
     outputs:
-    - contextPath: GoogleChatWebhook.CustomCard.Cards
```

Comment:
Same here.

Reply:
The GoogleChat API changed its outputs and no longer includes all of those values. We now receive the following output when interacting with the API. This is why I modified all of the outputs in this PR. 

`{
    "name": "spaces/AAAA5weu7HQ/messages/h229LcOBQTA.h229LcOBQTA",
    "text": "Hello from the python test client",
    "thread": {
        "name": "spaces/AAAA5weu7HQ/threads/h229LcOBQTA"
    },
    "space": {
        "name": "spaces/AAAA5weu7HQ"
    }
}`

Reply:
if so, lets add a breaking changes as you can see here https://xsoar.pan.dev/docs/documentation/release-notes#breaking-changes-version
In addition, please update the release notes to be more informative about the cause of the change (deleted outputs) and list that we also removed some outputs (list them).
Thank you!

Reply:
Done

Reply:
please do not remove the GoogleChatWebhook.CustomCard.Cards output as we do receive it in the response
thanks!